filebeat/generator/module/module.go

```diff
@@ -0,0 +1,33 @@
+package module
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/elastic/beats/filebeat/generator"
+)
+
+// Generate generates a new module.
+// If module exists, error is returned.
+func Generate(module, modulesPath, beatsPath string) error {
+	modulePath := path.Join(modulesPath, "module", module)
+	if generator.DirExists(modulePath) {
+		return fmt.Errorf("module already exists: %s", module)
+	}
+
+	err := generator.CreateDirectories(modulePath, []string{path.Join("_meta", "kibana", "6")})
```

Comment:
Should we make it configurable? Maybe passing a param using a flag e.g. `--kibana-version`? It could be 6 by default until we release version 7.

Reply:
Instead of having the module generator creating the correct directory, it perhaps should be the kibana exporting command that takes care of the directory creation. So we don't create the directories here but they are only add when someone actually exports dashboards. It would be mean we have to add some knowledge about modules and Kibana directories to the exported but I think that would make sense.

Reply:
I think it's a good idea.
I am wondering if there are use cases when a users would need directories without calling `export`. But I don't see any.